lib/PCAP/Bwa.pm

```diff
@@ -83,7 +94,7 @@ sub mem_setup {
   }
   # do some checking to ensure input BAM/CRAM hasn't been through mismatchQc
   # if it has check for use of at least bammaskflags
-  PCAP::Bam::mismatchQc_checks($options->{'raw_files'});
+  PCAP::Bam::mismatchQc_checks($options->{'raw_files'}) unless($skip_mmqc_check);
```

Comment:
If merging pre-generated lanes we don't want to block stuff data processed by mmQc. This step is reused by mapping and merging, hence the distinction